machine resource added (#2)
iotrek/slate: source/index.html.md

---
title: IoTrek

includes:
  - machine
  - errors

search: true
---

# Introduction

> Welcome to IoTrek\`s mortenson API every resource and api is listed here woth its request and response bodies and examples.

Reviewer: ```suggestion
Welcome to IoTrek\`s  API document. APIs are listed with their request, response and error bodies with examples.
```
